usecases/schema/class.go

```diff
@@ -69,7 +69,7 @@ func (h *Handler) GetCachedClass(ctxWithClassCache context.Context,
 			return nil, 0, err
 		}
 		if class == nil {
-			return nil, 0, nil
+			return nil, 0, ErrNotFound
```

Comment:
change(required): We need to update usages here to handle a class not found error. There are definitely places (auto schema for example) where we interpret a nil pointer as "class is missing" and continue on and create the class. This change would break that behaviour.